.travis.yml

```diff
@@ -7,14 +7,14 @@ dist: trusty
 
 matrix:
   include:
-  - go: 1.9
+  - go: 1.11.1
     env:
     - TESTS=true
     - COVERAGE=true
-  - go: 1.9
+  - go: 1.11.1
     env:
     - CROSSDOCK=true
-  - go: 1.9
+  - go: 1.11.1
```

Comment:
we need to have an entry for N-1 version (i.e. 1.10 right now) that runs the tests, but does not do linting / formatting (since those may change between versions).

Reply:
Could you tell me why N-1 version should be used for go-client?

Reply:
This is a library that runs in user application space, we need to worry about backwards compatibility so that we don't force them to upgrade Go version in sync with this lib. It doesn't mean we need to keep testing with many older versions, but testing with the previous (N-1) version is a common practice.

Reply:
Let's do it separately - #344

Reply:
Got it, thanks for your explanation :)